lib/scala/searcher/src/test/scala/org/enso/searcher/sql/SuggestionsRepoTest.scala

```diff
@@ -1698,6 +1723,32 @@ class SuggestionsRepoTest extends AnyWordSpec with Matchers with RetrySpec {
         ).flatten
     }
 
+    "search suggestion by the static attribute" taggedAs Retry in withRepo {
+      repo =>
+        val action = for {
+          _   <- repo.insert(suggestion.module)
+          _   <- repo.insert(suggestion.tpe)
+          _   <- repo.insert(suggestion.constructor)
+          id3 <- repo.insert(suggestion.method)
+          _   <- repo.insert(suggestion.conversion)
+          _   <- repo.insert(suggestion.function)
+          _   <- repo.insert(suggestion.local)
+          res <-
+            repo.search(
+              None,
+              Seq(),
+              None,
+              None,
+              None,
+              Some(true)
```

Comment:
For completeness, would be probably good to have a test that checks non-static methods only